src/testRunner/unittests/tsc/helpers.ts

```diff
@@ -118,7 +118,7 @@ namespace ts {
         sys.baseLine = () => {
             const patch = fs.diff(baseFs, { includeChangedFileWithSameContent: true });
             // eslint-disable-next-line no-null/no-null
-            Harness.Baseline.runBaseline(`${isBuild(commandLineArgs) ? "tsbuild" : "tsc"}/${scenario}/${buildKind || BuildKind.Initial}/${subScenario.split(" ").join("-")}.js`, patch ? vfs.formatPatch(patch) : null);
+            Harness.Baseline.runBaseline(`${isBuild(commandLineArgs) ? "tsbuild" : "tsc"}/${scenario}/${buildKind || BuildKind.Initial}/${subScenario.split(" ").join("-")}.js`, patch ? vfs.formatPatch(patch).replace(version, "CURRENT.VERSION") : null);

```

Comment:
Can you please change this to use fakes.version instead.. We do that in the buildinfo already. Thank you for catching this.

Reply:
Sure.

Reply:
Also better yet in the `serializeOutput in virtualFileSystemWithWatch`

Reply:
`tscCompile` uses `fakes.System`, and not `TestServerHost`; so modifying `serializeOutput` in `virtualFileSystemWithWatch` does nothing.

Reply:
I've modified `fakes.patchHostForBuildInfoReadWrite` to also override `sys.write` to replace the current version with `fakes.version`. I think that should move it closer to a more general position where the versions are handled in bundle infos like what you were thinking?